x-pack/solutions/security/plugins/security_solution/public/sourcerer/containers/use_init_sourcerer.tsx

```diff
@@ -240,14 +240,14 @@ export const useInitSourcerer = (
       );
     }
   }, [
-    activeTimeline,
+    activeTimeline?.savedObjectId,
```

Comment:
we only care about the id value, don't re call due to `activeTimeline` ref changes.